VMs/metrics/README.md

```diff
@@ -241,3 +314,21 @@ to set up the proxies for a number of programs, including, `apt`, `git`, and `do
 make this process simpler, an example [`user-data.proxy`](ubuntu16.04/user-data.proxy) file
 is included in the `ubuntu16.04` subdirectory.
 
+### Tuning `checkmetrics.toml`
+
+The default `checkmetrics.toml` file provided in each distro subdirectory with the VM
+scripts will no doubt match the results produced on your particular system. You will
```

Comment:
Since you're suggesting this file will need modification, should that be

> will no doubt not match

Reply:
Yarp, saw that when making another change - an I thought I'd tried quite hard to get that right. Changed to 'will probably not match'.